src/dtx/dtx_common.c

```diff
@@ -1853,6 +1855,8 @@ dtx_cont_register(struct ds_cont_child *cont)
 		D_GOTO(out, rc = -DER_NOMEM);
 	}
 
+	cont->sc_agg_eph_gap = d_sec2hlc(vos_get_agg_gap());
```

Comment:
I think it's not necessary to duplicate a constant value for each container.

Reply:
I will consider a dss variable to replace that.